docs/app/interfaces/IPlunk.ts

```diff
@@ -1,3 +1,13 @@
 export interface IPlunk {
```

Comment:
Looking at this again, we may need to change it to something like this:

```typescript
export interface IPlunk {
    files: {
        [key: string]: string;
    };
    modules?: {
        imports?: string[];
        library?: string;
    }[];
    mappings?: {
        alias: string;
        source: string;
    }[];
}
```

As we may want to import things from multiple libraries and the current interface doesn't allow for this